packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Image from 'next/image';
+import type { ImageProps } from 'next/image';
+import { useRouter } from 'next/router';
+
+interface ContentProps {
+	title: string;
+	imageProps?: ImageProps;
+	content: string;
+	date?: string;
+}
+
+/**
+ *
+ * @param props - The props needed for the ContentWithImage component
+ * @param props.title - The title of your content
+ * @param props.imageProps - All props the user wishes to pass to the next/image component
+ * @remarks
+ * imageProps is an optional prop to be used if there is an image to be associated with the content.
+ * If imageProps is used it is required that the user passes in values for src, width, and height
+ * See the documentation link below for more information on optional and required props
+ * @see {@link https://nextjs.org/docs/api-reference/next/image} for all next/image documentation
```

Comment:
Let's move the `@remarks` to the bottom of the TSDoc comment, after the `@returns`, and we can move the `@see {@link}` part into the `@param` comment for `props.imageProps`